scripts/pi-hole/php/FTL.php

```diff
@@ -6,12 +6,35 @@
 *  This file is copyright under the latest version of the EUPL.
 *  Please see LICENSE file for your rights under this license. */
 
+function piholeFTLConfig()
+{
+	global $piholeFTLConfig;
```

Comment:
A global variable with this name does not seem to exist anywhere in the code, currently.

Reply:
I haven’t used globals with PHP previously, but by my understanding this would cause the value of $piholeFTLConfig to be remembered after the function is executed the first time, causing the below isset() to match, and avoiding unnecessary re-reads of the config file if the settings would be used in more than one place. Please correct if I understood how it works wrong.

Personally I would use a class to handle settings, but there seems not to be that kind of convention in AdminLTE, so I didn’t go to invent yet another way to save globally accessible data. If you have an idea how to implement this in better way, I can modify my code.

Reply:
In my understanding of the PHP language, you should use a `static` rather than a `global` variable to achieve what you want.

There aren't all that many conventions in this repository as our web interface includes a lot of external contributions and we do not enforce all that many rules on external contributors. Everything that is clean and and works solid is typically accepted by us. You're free to do what think is appropriate.

Reply:
Looks like you’re correct. I’ll change the code to use static instead. 

I’d rather not add yet another new big thing to the repo unless there is consensus that it would be useful for others too, since it would in the end just get in the way.